compiler/modules/front/codeGen.bal

```diff
@@ -1216,6 +1227,98 @@ function codeGenExpr(CodeGenContext cx, bir:BasicBlock bb, Environment env, s:Ex
     panic err:impossible("unrecognized expression type in code gen: " +  s:exprToString(expr));
 }
 
+type LiftResult record {|
+    bir:Operand[] operands;
+    bir:BasicBlock nextBlock;
+    bir:BasicBlock? nilBlock;
```

Comment:
nextBlock and nilBlock are not a good pair because `next` and `nil` do not have the same relationship to `Block`: `ifNilBlock` better.

Reply:
fixed